artifactory.py

```diff
@@ -100,26 +100,29 @@ def read_config(config_path=default_config_path):
     result = {}
 
     for section in p.sections():
-        username = (
-            p.get(section, "username") if p.has_option(section, "username") else None
-        )
-        password = (
-            p.get(section, "password") if p.has_option(section, "password") else None
-        )
-        verify = (
-            p.getboolean(section, "verify") if p.has_option(section, "verify") else True
-        )
-        cert = p.get(section, "cert") if p.has_option(section, "cert") else None
+        username = p.get(section, "username", fallback=None)
+        password = p.get(section, "password", fallback=None)
+
+        try:
+            verify = p.getboolean(section, "verify", fallback=True)
+        except ValueError:
+            # the path to a CA_BUNDLE file or directory with certificates of trusted CAs, see #281
```

Comment:
>see #281
If you mention an issue - provide the full link to the issue. It's be useful for forks, for instance.

Reply:
and save developers time :)

Reply:
@allburov 
agreed, done